src/tracing/tracing.ts

```diff
@@ -55,6 +55,7 @@ export class Tracing implements TelemetryBase<void> {
       instrumentations: [
         ...(getNodeAutoInstrumentations({
           ...this.autoInstrumentationsConfigMap,
+          // eslint-disable-next-line @typescript-eslint/naming-convention
           '@opentelemetry/instrumentation-pino': { enabled: false },
```

Comment:
```suggestion
          ['@opentelemetry/instrumentation-pino']: { enabled: false },
```
With this you dont need the eslint-ignore.